src/main/java/nextstep/subway/applicaion/dto/PathResponse.java

```diff
@@ -21,13 +23,22 @@ public PathResponse(List<StationResponse> stations, int distance, int duration)
         this.duration = duration;
     }
 
+    public PathResponse(List<StationResponse> stations, int distance, int duration, int fare) {
+        this.stations = stations;
+        this.distance = distance;
+        this.duration = duration;
+        this.fare = fare;
+    }
+
     public static PathResponse of(Path path) {
         List<StationResponse> stations = path.getStations().stream()
                 .map(StationResponse::of)
                 .collect(Collectors.toList());
         int distance = path.extractDistance();
+        int duration = path.extractDuration();
+        int fare = SubwayFare.calculateFare(distance);
```

Comment:
요구사항을 잘못 이해하신 것 같아요 😢 
요금은 조회한 경로의 거리가 아닌 최단 거리로 계산되어야 하는데요 
<img width="493" alt="image" src="https://github.com/next-step/atdd-subway-fare/assets/8064743/cabcb881-7bf5-4b81-8a14-c8c335c97678">

요금을 계산할 때 최단 거리 값이 필요하고 거리 값은 path가 알 수 있으니 책임을 위임하면 어떨까요?
```suggestion
        int fare = path.fare();
```

Reply:
최소 시간 기준으로 경로 조회하더라도 최단 거리 기준으로 요금 계산을 하도록 수정했습니다! 
현재는 최단 거리 기준으로 ShortestDistancePath를 한 개 더 생성하였는데 PathType을 최소 시간 기준으로 경로 조회를 하는 경우에도 최단 거리 값을 알 수 있는 방법이 있을까요??